### DISCUSSION TITLE
Adult+ Content Delay In Pop Up?

### DISCUSSION THREAD

**Discussion by jwag12787** [Tetrakern/fictioneer#64] [Q&A] [answered]
When utilizing the Adult+ content prompt, I notice there is a delay in the pop up.  Is there a way to reduce this delay?  It does allow for a small period of time for navigation through the page.  Also, is it possible to show, but blur, the cover art for these titles in the main stories shortcode?

**Comment by jwag12787** [comment]
I guess in follow up, is there a way to set it so that users can select/unselect "tags" in their profile to auto-hide from a main library that lists all books?  Or do I just need to make a mental note to keep different libraries separated intentionally when building out?

**Comment by Tetrakern** [comment, marked as answer]
> When utilizing the Adult+ content prompt, I notice there is a delay in the pop up. Is there a way to reduce this delay? It does allow for a small period of time for navigation through the page. Also, is it possible to show, but blur, the cover art for these titles in the main stories shortcode?

The age confirmation has a 2.5 seconds timeout in order to not impact search engine crawlers (with JS), because blocking a page isn't particularly great for your ranking. I also check the user agent for that, but that's not reliable. I assumed that 2.5 seconds is not enough to read a significant part of a page, not that the modal actually stops anyone. You cannot change the delay, but I think I'll just remove ist.

As for your other question, there is no such feature. This could certainly be done, but not for free.
